distribution/openrct2.d.ts

```diff
@@ -1558,10 +1558,14 @@ declare global {
         direction: Direction;
         object: number;
         primaryColour: number;
+        /** If the element is a banner, this is the text colour. */
         secondaryColour: number;
         tertiaryColour: number;
-        bannerIndex: number | null;
         slope: Direction;
+        /** Writing to bannerIndex is @deprecated and may result in uncontrolled behaviour. */
```

Comment:
The `@deprecated` should be at the front for it to be a proper JSDoc comment, see [here](https://jsdoc.app/tags-deprecated).

Reply:
I don't want to start the comment with @deprecated: the IDE will mark the whole property as deprecated and not only writing to it.

Reply:
Then I think it's better we don't use the `@deprecated` at all and just mark it read-only.

Reply:
Why?

Reply:
Because it's incorrect usage of the JSDoc tag. I meant you can just remove the `@` e.g. `@deprecated` -> `deprecated`.

Reply:
Ah, that makes more sense than just removing the whole comment.